json/DataTypes/DecodeStudentLong.roc

```diff
@@ -0,0 +1,81 @@
+module [getListStudents]
+
+import JsonData exposing [Json]
+import Student exposing [Student, Module]
+
+JsonErrors : [
+    FieldNotFound Str,
+    ExpectedJsonObject Str,
+    ExpectedJsonArray Str,
+]
+
+getListStudents : Json -> Result (List Student) JsonErrors
+getListStudents = \json ->
+    when json is
+        Object obj ->
+            when Dict.get obj "students" is
+                Ok students -> checkStudentList students
+                _ -> Err (FieldNotFound "Expected field with name students in object")
+
+        _ -> Err (ExpectedJsonObject "Expected an Object")
```

Comment:
This is a great use of `_ ->` because we really do want to ignore all the other possibilities, no matter what they are. 💯